.devcontainer/postCreateCommand.sh

```diff
@@ -9,7 +9,7 @@ cd "${SCRIPT_DIR}"
 GIT_ROOT="$(git rev-parse --show-toplevel)"
 
 echo "Setting up environment variables by copying .env files"
-cp -n "${GIT_ROOT}/packages/server/.env-example" "${GIT_ROOT}/packages/server/.env" || true
+cp -n "${GIT_ROOT}/packages/server/.env.example" "${GIT_ROOT}/packages/server/.env" || true
```

Comment:
it always annoyed me that it was .env-example in `server`

docs change PR: https://github.com/specklesystems/speckle-docs/pull/281